### ISSUE TITLE
Generate ER Diagramm for GitBucket itself.

### ISSUE THREAD

**Issue by aadrian** [nus/gitbucket-plantuml-plugin#14]
Please generate an ER Diagram for GitBucket DB itself, and display it in Admin UI.

Initially requested here: https://github.com/gitbucket/gitbucket/issues/765

but since PlantUML supports ER Diagram rendering now: https://github.com/plantuml/plantuml/issues/39

it would be much better if this plug-in would generate the diagram automatically, so no need for manual work everytime the DB Model is changing (e.g. new versions, or installed plug-ins change the DB). 

Thank you.

**Comment by nus** [comment]
I think that the ERD file is generated by [Amateras Modeler](https://github.com/takezoe/amateras-modeler).
If one plugin display GitBucket DB in AdminUI, the plugin must be for rendering ERD files.

**Comment by aadrian** [comment]
> I think that the ERD file is generated by Amateras Modeler.

Yes, it's a manual work for now, it's not very usable and it does not reflect the running instance.

> If one plugin display GitBucket DB in AdminUI, the plugin must be for rendering ERD files.

Not sure I understand what you mean, but:
 - this plug-in can already render a PlantUML diagram (any kind) - e.g. ```*.pu``` or ```*.puml``` files
 - PlantUML already supports ER Diagrams: https://github.com/plantuml/plantuml/issues/39

So what would be needed would be:
 1. have an admin config for PlantUML specific settings - only nice to have for usability
 1. have an "on the fly" generated PlantUML ER diagram file from the running DB
 1. have another Tab in the settings where to display this diagram to.
